### DISCUSSION TITLE
Como podria utilizar adapter?

### DISCUSSION THREAD

**Discussion by maribelcalla** [JosephCalla/Design-Patterns-Swift#7] [Q&A] [answered]
Estado buscando otra documentacion ,hasta ahora no he  comprendido el  patron Adapter

**Comment by JosephCalla** [comment, marked as answer]
Take a look this https://github.com/JosephCalla/Design-Patterns-Swift, @maribelcalla

**Comment by maribelcalla** [comment]
Thanks Joseph

**Comment by JosephCalla** [comment]
No problem 👍
